corehq/apps/app_manager/models.py

```diff
@@ -2523,6 +2522,14 @@ def get_form(self, i):
         except IndexError:
             raise FormNotFoundException()
 
+    def get_form_index(self, unique_id):
+        for index, form in enumerate(self.get_forms()):
+            if form.unique_id == unique_id:
+                return index
+        error = _("Could not find form with ID='{unique_id}' in module '{app_name}'.").format(
+            app_name=self.name, unique_id=unique_id)
```

Comment:
Is `app_name` the name of the app or the module? The error text makes it sound like a module name, but the variable name makes it sound like an app name. It looks like either the variable or the error text could be changed to clear that up. Guessing from the code below in `get_module_index` that the error text should be changed.

Reply:
Yup, should be `module_name`